.github/workflows/nightly_ondemand.yml

```diff
@@ -216,7 +216,7 @@ jobs:
           hf_token: ${{ secrets.HUGGING_FACE_HUB_TOKEN }}
       - name: Summarize archieve files
         id: summary
-        if: always()
+        if: ${{ ! cancelled() }}
```

Comment:
Why we need this change?

Reply:
if set always, it will be always run even if cancelled.